### ISSUE TITLE
XFA PDF RadioButton custom reader text ignored

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3079]
**Reported by BR001 on 2013-03-12 07:04**
The custom reader text assigned to a radio button group (An Exclusive Group in XFA speak) is ignored by NVDA.

Instructions on adding this text are available at http://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/PDF10#working-examples/PDF10#working-examples/PDF10 but NVDA seems to ignore them.

**Comment by bhavyashah** [comment]
While this report/request is definitely technical in nature, I still feel like I should be able to make some heads or tails of it, which, unfortunately, I don't. Is this a PDF specific bug report pertaining to Adobe Reader, an application specific issue with a certain program, or a general document/web elements related deficiency in NVDA? I don't believe we can contact the original author of this ticket (or maybe we can, I am not sure). Therefore, @feerrenrut, do you believe we have sufficient information to work on this ticket in the future, or is the scope of the same unclear?

**Comment by feerrenrut** [comment]
I think the link provides enough information to be able to reproduce the issue, but unfortunately I do not have Adobe Acrobat Pro which could be used to provide a test case. I would prefer to leave this open, hopefully someone is able to produce a PDF using the linked instructions and test this.

**Comment by Adriani90** [comment]
@leonardder do you have Acrobat Pro maybe at Babbage and could please try to reproduce it?
@BR001 can you still reproduce it?

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author on this since many years, closing as abandoned. Please comment on it with an example pdf where we can reproduce this issue or with more testing details with newest NVDA versions and Adobe Acrobat, and then we can reopen.
